aws/master-cft/paid-existing-ocp/cft-mas-core-dev.json

```diff
@@ -696,7 +696,7 @@
                                 "git clone https://github.com/ibm-mas/ansible-devops.git;",
                                 "cd ansible-devops;",
                                 "rm -rf multicloud-bootstrap 1>/dev/null 2>&1;",
-                                "git clone https://github.com/ibm-mas/multicloud-bootstrap.git;",
+                                "git clone -b aws-paid https://github.com/ibm-mas/multicloud-bootstrap.git;",
```

Comment:
Hi @shajeena  Is this change checked in by mistake? We should not pull from the specific branches.

Reply:
It is needed temporarily. Now removed it from template. We'll add it while triggering